code/datums/status_effects/debuffs/drugginess.dm

```diff
@@ -12,7 +12,7 @@
 	RegisterSignal(owner, COMSIG_LIVING_DEATH, PROC_REF(remove_drugginess))
 
 	owner.add_mood_event(id, /datum/mood_event/high)
-	owner.overlay_fullscreen(id, /atom/movable/screen/fullscreen/high)
+	// owner.overlay_fullscreen(id, /atom/movable/screen/fullscreen/high) BUBBERSTATION CHANGE: REMOVES HIGH OVERLAY.
```

Comment:
If you hate it that much, code it to be a toggleable game preference.

Reply:
way too much effort my dude for something that is annoying